plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerQueryBuilder.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.sqlserver;
+
+import com.google.common.base.Joiner;
+import io.trino.plugin.jdbc.DefaultQueryBuilder;
+import io.trino.plugin.jdbc.JdbcClient;
+import io.trino.plugin.jdbc.JdbcProcedureHandle.ProcedureQuery;
+import io.trino.plugin.jdbc.logging.RemoteQueryModifier;
+import io.trino.plugin.jdbc.ptf.Procedure.ProcedureInformation;
+import io.trino.spi.connector.ConnectorSession;
+
+import javax.inject.Inject;
+
+import java.sql.Connection;
+
+public class SqlServerQueryBuilder
+        extends DefaultQueryBuilder
+{
+    @Inject
+    public SqlServerQueryBuilder(RemoteQueryModifier queryModifier)
+    {
+        super(queryModifier);
+    }
+
+    @Override
+    public ProcedureQuery createProcedureQuery(JdbcClient client, ConnectorSession session, Connection connection, ProcedureInformation procedureInformation)
+    {
+        return new ProcedureQuery(
+                "EXECUTE %s.%s %s"
+                        .formatted(
```

Comment:
it looks like one could inject some SQL by using inputs.

Reply:
Thanks for pointing it out. Have added additional checks.